src/main/java/com/saebyeok/saebyeok/service/ArticleService.java

```diff
@@ -117,4 +118,8 @@ public ArticleResponse readMemberArticle(Member member, Long articleId) {
                 }).
                 collect(Collectors.toList());
     }
+
+    public List<Article> getLast1MonthArticles(Member member) {
+        return articleRepository.findAllByMemberAndCreatedDateGreaterThanEqualAndIsDeleted(member, LocalDateTime.now().minusDays(ANALYSIS_LIMIT_DAYS), false);
+    }
```

Comment:
ArticleService에 ANALYSIS_LIMIT_DAYS라는 상수가 있는 게 맞을까? 라는 생각이 들었어요! 
사용하는 부분이 더 있다면 고민해봤을텐데 일단 사용처가 이 메서드밖에 없구,
AnalysisService에 이미 같은 의미의 INQUIRY_DAYS라는 상수가 존재하고 있어서요~
(이제 보니 INQUIRY_DAYS라는 상수명이 좀 추상적이당 😅 요 상수명을 ANALYSIS_LIMIT_DAYS로 바꾸는 게 더 명확하겠어요!)
상수의 위치 자체도 컨텍스트 상 AnalysisService에 존재하는게 자연스러운 것 같아서
AnalysisService에서 파라미터로 기준 날짜를 건네주는 게 더 좋지 않을까 하는데 어떻게 생각하쉴지 🤩

또 메서드명에 Last1Month라는 AnalysisService의 직접적인 요구사항? 정책에 깊게 연관된 표현보다는, 
멤버와 제한된 기간에 속한 Article만 찾아서 준다는 기능이 표현되는 게 낫지 않을까 싶네요! 
일단 이 메서드는 ArticleService이기도 하고, 차후 ANALYSIS_LIMIT_DAYS가 30에서 다른 숫자로도 바뀔 수 있으니까요 ㅎㅎ
getLimitedDaysArticles..? 썩 마음에 들진 않지만 그나마 떠올리자면 요런 식? 😂

```suggestion
    public List<Article> getLimitedDaysArticles(Member member, int limitDays) {
        return articleRepository.findAllByMemberAndCreatedDateGreaterThanEqualAndIsDeleted(member, LocalDateTime.now().minusDays(limitDays), false);
    }
```

Reply:
좋은 아이디어 같아요! 반영했습니다!!